const { ProductCategory, ProductItem } = require('../models');
const createError = require('../utils/createError');

exports.productCategory = async (req, res, next) => {
  try {
  } catch (err) {
    next(err);
  }
};

exports.productItem = async (req, res, next) => {
  try {
  } catch (err) {
    next(err);
  }
};

exports.productComment = async (req, res, next) => {
  try {
  } catch (err) {
    next(err);
  }
};
exports.productRating = async (req, res, next) => {
  try {
  } catch (err) {
    next(err);
  }
};

exports.cart = async (req, res, next) => {
  try {
  } catch (err) {
    next(err);
  }
};

exports.orderHistory = async (req, res, next) => {
  try {
  } catch (err) {
    next(err);
  }
};
